src/i18n/strings/en_EN.json

```diff
@@ -1767,6 +1770,32 @@
     "%(brand)s is end-to-end encrypted, but is currently limited to smaller numbers of users.": "%(brand)s is end-to-end encrypted, but is currently limited to smaller numbers of users.",
     "You do not have sufficient permissions to change this.": "You do not have sufficient permissions to change this.",
     "Call type": "Call type",
+    "Email summary": "Email summary",
+    "Receive an email summary of missed notifications": "Receive an email summary of missed notifications",
+    "Select which emails you want to send summaries to. Manage your emails in <button>General</button>.": "Select which emails you want to send summaries to. Manage your emails in <button>General</button>.",
+    "People, Mentions and Keywords": "People, Mentions and Keywords",
+    "Mentions and Keywords only": "Mentions and Keywords only",
+    "Switch now": "Switch now",
+    "<strong>Update:</strong> We have updated our notification settings. This won’t affect your previously selected settings.": "<strong>Update:</strong> We have updated our notification settings. This won’t affect your previously selected settings.",
+    "Show message preview in desktop notification": "Show message preview in desktop notification",
+    "I want to be notified for (Default Setting)": "I want to be notified for (Default Setting)",
+    "This setting will be applied by default to all your rooms.": "This setting will be applied by default to all your rooms.",
+    "Play a sound for": "Play a sound for",
```

Comment:
I'm not sure this is a good i18n, it should include the component containing the options so that it can be re-ordered depending on the exact language requirements